centreon/src/Core/Dashboard/Infrastructure/API/ShareDashboard/ShareDashboardRoute.yaml

```diff
@@ -0,0 +1,7 @@
+ShareDashboardPlaylist:
+  methods: PUT
+  path: /configuration/dashboards/{dashboardId}/shares
+  requirements:
+    dashboardId: '\d+'
+  controller: 'Core\Dashboard\Infrastructure\API\ShareDashboard\ShareDashboardController'
+  condition: "request.attributes.get('version') >= 23.10"
```

Comment:
```suggestion
  condition: "request.attributes.get('version') >= 24.10"
```

Reply:
23.10 is correct as this new endpoint is also backported on 23.10